cfgmgr/portmgr.cpp

```diff
@@ -176,3 +204,9 @@ bool PortMgr::writeConfigToAppDb(const std::string &alias, const std::string &fi
 
     return true;
 }
+
+bool PortMgr::writeConfigToAppDb(const std::string &alias, std::vector<FieldValueTuple> &field_values)
+{
+    m_appPortTable.set(alias, field_values);
+    return true;
```

Comment:
do we need the return value?

Reply:
The other func (writeConfigToAppDb) has one. To keep it consistent, i added it here